webv2/web.go

```diff
@@ -656,6 +673,24 @@ func setParentTable(w http.ResponseWriter, r *http.Request) {
 	}
 	tableInterleaveStatus := parentTableHelper(table, update)
 
+	if tableInterleaveStatus.Possible {
```

Comment:
After removing a column from primary key after interleaving the table it still shows as interleaved : https://drive.google.com/file/d/1m5Rj2VVCWE0pCCsnTqq8SDITafzWcuRQ/view?usp=sharing&resourcekey=0-xCKQPLqlQ8SeZo81oTkzEQ

Reply:
After we convert a table to interleaved and remove a column from the primary key of the interleaved table , the status still shows as interleaved.

Reply:
Solved